src/components/AuthLayout.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+
+import './auth-layout.css';
+
+const AuthLayout = ({ name, children }) =>
+    (
+        <section>
+            <h1>{name}</h1>
+            <article id="auth-layout">
```

Comment:
This element does not represent the auth layout, this represents an article in the auth layout, apply the id to the correct element.